### PR TITLE
wander: init at 0.4.1

### PR DESCRIPTION
**Pull request by astro**
###### Description of changes

Provides an alternative to `damon`

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by nrdxp** [comment]
Result of `nixpkgs-review pr 181030` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>wander</li>
  </ul>
</details>

File: pkgs/tools/admin/wander/default.nix [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    sha256 = "sha256-Z3jvKBhzlATTh6zPoJoMmg/DAE5/Ur3Tb3sdgGPEm6k=";
+  };
+
+  vendorSha256 = "sha256-gWQ8GbtghhCRq6tOU6qmWBuponmfUkUDAk3+dPtmMiE=";
+
+  meta = with lib; {
+    description = "Terminal app/TUI for HashiCorp Nomad";
+    license = licenses.mit;
+    homepage = "https://github.com/robinovitch61/wander";
+    maintainers = teams.c3d2.members;
```
Comment (nrdxp) [inline pkgs/tools/admin/wander/default.nix:20]: has everyone on this team agreed to be a maintainer for this?

Comment (revol-xut) [reply pkgs/tools/admin/wander/default.nix:20]: considering we want to use nomad at some point for our infrastructure I would say yes.

Comment (SuperSandro2000) [reply pkgs/tools/admin/wander/default.nix:20]: > has everyone on this team agreed to be a maintainer for this?

The c3d2 team is for the hackspace with the same name and collects all software we package for our infrastructure. Before the team was created we kept packages under several different personal accounts which made it harder to keep track what we need to maintain collectively. It's fine for me (and probably the others) to add new packages that are/will be used in our infrastructure and I don't think we need the approval of each person in the team each time a new package is added.

**Pull request merged**